apps/avifdec.c

```diff
@@ -59,7 +53,7 @@ static void syntax(void)
     avifPrintVersions();
 }
 
-MAIN()
+int main(int argc, char * argv[])
```

Comment:
I guess we use `char*argv[]` for C and `char**argv` for C++?

Reply:
I don't know of such a difference between C and C++. I always use `char * argv[]`, but in this pull request I took care to go back to the original code, which exists for avifdec.c, avifenc.c, and are_images_equal.cc.

I don't know why I chose `char ** argv` for avifgainmaputil.cc. I may have copied it from are_images_equal.cc without thinking.